File: fibonacci.java
// Implementation of function to get the n-st number of a Fibonnaci sequence in Java

class Fibonnaci {
    public static int fibo(int n) {
        if (n <= 1) {
            return n;
        } else {
            return (fibo(n-1) + fibo(n-2));
        }

    }

    public static void main(String[] args) {
        System.out.println("Fibonacci Sequence\n");

        // Print the 20 first numbers of the Fibonacci Sequence
        for (int i = 0; i <= 20; i++) {
            System.out.println("index - " + i + "\t value = " + fibo(i));
        }
    }
}

File: fibonacci.py
# Implementation of function to get the n-st number of a Fibonnaci sequence in Python 

def fibo(n):
    if n <= 1:
        return n
    else:
        return (fibo(n-1) + fibo(n-2))

if __name__ == '__main__':
    print('Fibonacci Sequence\n')

    # Print the 20 first numbers of the Fibonacci Sequence
    for x in range(21):
        print('index - {} \t value = {}'.format(x, fibo(x)))